Makefile

```diff
@@ -68,7 +73,7 @@ dependencies:
 
 # release builds and installs release binaries.
 release: dependencies test-long
-	go install -a ./...
```

Comment:
Is there a reason that `release` and `xc` need to be running `test-long` anyway?

Okay, I actually know the reason, but I don't think it's terribly good one. And you'd need to add `REBUILD` regardless.